atc/api/accessor/role_action_map.go

```diff
@@ -17,6 +17,7 @@ var requiredRoles = map[string]string{
 	atc.GetCheck:                      "viewer",
 	atc.GetBuildPlan:                  "viewer",
 	atc.CreateBuild:                   "member",
+	atc.RerunJobBuild:                 "member",
```

Comment:
Heey,

I'm not familiar with this part of the code, but, should we then increase the coverage in the accessor tests?

https://github.com/concourse/concourse/blob/991453c585ea542fe8ab76fd5f7ad22002fdfbb9/atc/api/accessor/accessor_test.go#L440-L450

thanks!

Reply:
I just added a test for ensuring that all endpoints have a role assigned to it! It actually found a few that didn't have a role lol